runtime-manager/modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -5,17 +5,21 @@ endif::[]
 :page-aliases: general:getting-started:deploy-to-server.adoc, deploying-to-pcf.adoc
 
 
-image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-cloud2-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub 2.0"]
+image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub 1.0"]
 image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
 image:logo-server-active.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
 image:logo-rtf-active.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
 
 
 When you run an app in Anypoint Studio, the app deploys to an embedded test server in Studio. Because this server is not meant for production deployment and uptime restrictions apply, deploy your Mule app using one of the deployment options supported by Anypoint Runtime Manager.
 
-* *<<CloudHub Deployments,CloudHub>>*
+* *<<CloudHub 2.0 Deployments,CloudHub 2.0>>*
```

Comment:
Is this link working for you? In my local build, the URL looks like: runtime-manager/deployment-strategies.html#CloudHub%202.0%20Deployments

I think you need to create a target for the image links in this format: 
`[[cloudhub-deployments]]`

Reply:
The previous page had a section per each deployment option. I changed that and added the tables focused on the RTM features, but then I forgot to remove the references. I'll remove them, or find out where (and if) we want these to point to.